frontend/src/app/organization/organization.resolver.ts

```diff
@@ -48,8 +44,17 @@ export const organizationDetailResolver: ResolveFn<Organization | undefined> = (
       heel_life: '',
       public: false,
       events: null
-    };
+    }
   }
+
+  // Otherwise, return the organization.
+  // If there is an error, return undefined
+  return inject(OrganizationService).getOrganization(route.paramMap.get('slug')!)
+      .pipe(catchError((error) => {
+        console.log(error)
```

Comment:
Should this be removed?

Reply:
I actually added this `console.log` to this so that there is some diagnostic showing the error that occurs! Otherwise, pretty sure it just returns undefined - but, I can remove this or improve it if it does not follow best practices

Reply:
We need to identify what the best, app-wide practice is for surfacing errors easily. Fine for now.